src/Diagnostics.RuntimeHost/Controllers/DiagnosticControllerBase.cs

```diff
@@ -790,40 +790,7 @@ private async Task<IEnumerable<DiagnosticApiResponse>> ListDetectorsInternal(Run
             if (context.ClientIsInternal)
             {
                 dataProvidersMetadata = GetDataProvidersMetadata(dataProviders);
-            }
-
-            var changeSetId = queryParams.Where(s => s.Key.Equals("changeSetId")).Select(pair => pair.Value);
```

Comment:
QQ Rekha - will applens change analysis work after this ?

Reply:
Yes Shekhar, because we are using forms instead of these named query string parameters